extensions/smallrye-health/deployment/src/main/java/io/quarkus/smallrye/health/deployment/SmallRyeHealthUIConfig.java

```diff
@@ -16,12 +18,7 @@
      * Always include the UI. By default this will only be included in dev and test.
      * Setting this to true will also include the UI in Prod
      */
-    @ConfigItem(defaultValue = "false")
-    boolean alwaysInclude;
+    @ConfigItem
+    Optional<Boolean> alwaysInclude;
```

Comment:
Again, let's avoid this. It has a default value, it needs to be clear to the users.

Reply:
See comment above on this. Resolving here, let's discuss there, and once we have a way I'll update all